hawc/apps/lit/templates/lit/_reference_tag_conflict.html

```diff
@@ -13,36 +21,40 @@
     <p class="abstract abstract_collapsed cursor-pointer mb-1" title="Click to expand/collapse">{{ref.abstract|safe}}</p>
     <div class="row">
         <div class="col-md-12 pt-1">
-            <p class="mb-0"><i>Tags currently applied:</i></p>
-            {% for tag in ref.tags.all %}
-                {% include 'lit/_nested_tag.html' with tag=tag extra_classes='' %}
-            {% empty %}
-                <span><i>&lt;none&gt;</i></span>
-            {% endfor %}
-            {% for user_tag in ref.user_tags.all %}
-            <div class="tag-conf-item {% if forloop.first %} mt-1 {% endif %}">
-                <form>
-                    <div>
-                        <button class="approve btn btn-sm btn-success mr-2 align-center mb-2 mt-1"
-                            hx-post="{% url 'lit:api:reference-resolve-conflict' ref.pk %}">
-                            <i class="fa fa-fw fa-check" aria-hidden="true"></i>&nbsp;Approve</button>
-                        <b class="mb-0">{{user_tag.user.get_full_name}}</b> <i class="m-0 small">{{user_tag.user.email}}</i>
-                        <input name="user_tag_id" value={{user_tag.pk}} class="hidden">
-                    </div>
-                    {% with user_tag.consensus_diff as diff %}
-                        {% for tag in ref.tags.all %}
-                            {% if tag in diff.intersection %}
-                                {% include 'lit/_nested_tag.html' with tag=tag extra_classes='' %}
-                            {% else %}
-                                {% include 'lit/_nested_tag.html' with tag=tag extra_classes='refUserTagRemove' %}
-                            {% endif %}
-                        {% endfor %}
-                        {% for tag in diff.difference %}
-                            {% include 'lit/_nested_tag.html' with tag=tag extra_classes='refUserTag' %}
-                        {% endfor %}
-                    {% endwith %}
-                </form>
+            {% with ref.tags.all as tags%}
+            {% if tags %}
+            <div class="tag-conflict">
```

Comment:
changed some CSS here to horizontally align the existing consensus tags with the suggested tags; makes it easier to review